source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -5286,6 +5286,10 @@ export class Container extends StylableCardElementContainer {
         return this.getValue(Container.backgroundImageProperty);
     }
 
+    set backgroundImage(value: BackgroundImage){
```

Comment:
No, the lack of setter is by design. The `backgroundImage` property is always initialized to a BackgroundImage instance automatically (if that's not the case, it might be a regression and that would need to be fixed). Developers set `container.backgroundImage.url` etc. but never `container.backgroundImage`.